### PR TITLE
 #60 Display issue with Visual Studio 2022 17.12

### PR DESCRIPTION
**Pull request by serge-linnet**
PR adds configuration for the `outlining.chevron` element added in Visual Studio 2022 17.12
frappe
latte
mcc
mocha

### CODE REVIEW

**Comment by tayopi** [comment]
Thank you so much for doing this. It's been driving me crazy, but I didn't know how to fix it.

**Review by djflan — APPROVED** [review]

**Comment by djflan** [comment]
no longer a maintainer on this project - but the changes here look good and I think would solve the issue

**Comment by tayopi** [comment]
> no longer a maintainer on this project - but the changes here look good and I think would solve the issue

Do you know who the maintainers are?

**Comment by djflan** [comment]
@tayopi there isn't one at the moment

**Comment by sgoudham** [comment]
Yeah, I'm happy to merge if you are.
I'll need to implement an automated deployment process similar to ctp/vscode at some point.

**Comment by wbpayne22902** [comment]
@sgoudham I have no issues at all.  Built my own VSIX from the PR #61 and am using it and having no issues whatsoever in VS 17.12.2.

**Comment by sgoudham** [comment]
> @sgoudham I have no issues at all. Built my own VSIX from the PR #61 and am using it and having no issues whatsoever in VS 17.12.2.

Great, thanks for confirming again. Might be a silly question but could you use please tell me how you built the VSIX? I'm guessing you need to open this project in Visual Studio and build it from there?

**Comment by wbpayne22902** [comment]
@sgoudham Yes, you open the solution from the PR in Visual Studio and build it. You'll likely need to have the Visual Studio Extension Development workload installed if you don't already have it.

**Comment by 3rdey3** [comment]
when will a new release be made for this change?

**Comment by sgoudham** [comment]
Sorry, I haven't had the time for OSS as of late @3rdey3. I'll try my best at the weekend to get a release out.

**Pull request merged**
